core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/BlockDataMap.java

```diff
@@ -297,11 +299,17 @@ private DataMapRowImpl loadBlockMetaInfo(CarbonRowSchema[] taskSummarySchema,
     // min max flag for task summary
     boolean[] taskSummaryMinMaxFlag = new boolean[segmentProperties.getColumnsValueSize().length];
     Arrays.fill(taskSummaryMinMaxFlag, true);
+    LOGGER.warn("GGGGGGGGGGG KKKKKK BBBBBBB");
+    for (Map.Entry<String, BlockMetaInfo> entry : blockletDataMapInfo.getBlockMetaInfoMap()
+        .entrySet()) {
+      LOGGER.warn(entry.getKey());
+    }
     long totalRowCount = 0;
     for (DataFileFooter fileFooter : indexInfo) {
       TableBlockInfo blockInfo = fileFooter.getBlockInfo().getTableBlockInfo();
       BlockMetaInfo blockMetaInfo =
           blockletDataMapInfo.getBlockMetaInfoMap().get(blockInfo.getFilePath());
+      LOGGER.warn("loadBlockMetaInfo OOOO " + blockInfo.getFilePath());
```

Comment:
please give a decent log message, or make it debug

Reply:
removed